src/apps/bm_devkit/hello_world/CMakeLists.txt

```diff
@@ -24,11 +24,6 @@ set(BRISTLEMOUTH_INCLUDES
     ${SRC_DIR}/lib/middleware/services
     ${SRC_DIR}/third_party/
     ${SRC_DIR}/third_party/aligned_malloc
-    ${SRC_DIR}/lib/bm_core/include
-    ${SRC_DIR}/lib/bm_core/bm_common_messages
-    ${SRC_DIR}/lib/bm_core/third_party/tinycbor/src
-    ${CMAKE_BINARY_DIR}/_deps/bm_common-src/include
-    ${CMAKE_BINARY_DIR}/_deps/bm_common-src/third_party/crc
```

Comment:
Updates to bm_core allows us to not have to define these includes in application CMake files.